pkg/utils/test/clients.go

```diff
@@ -42,6 +44,11 @@ func NewK8sClients(env environments.Environment) (K8sClients, error) {
 	if err != nil {
 		return clients, err
 	}
+	clients.KICClient, err = kicclient.NewForConfig(env.Cluster().Config())
```

Comment:
Please consider switching to `kubernetes-configuration` client instead for manipulating `KongPlugin`s.

Reply:
Very good idea, I'm doing it